raiden-cli/src/routes/pendingTransfers.ts

```diff
@@ -1,20 +1,61 @@
 import { Router, Request, Response } from 'express';
+import { timer } from 'rxjs';
+import { filter, takeUntil, map, toArray } from 'rxjs/operators';
+import { RaidenTransfer, isntNil } from 'raiden-ts';
+
 import { Cli } from '../types';
+import { validateOptionalAddressParameter } from '../utils/validation';
 
-export function makePendingTransfersRouter(this: Cli): Router {
-  const router = Router();
+enum ApiRole {
```

Comment:
So far we put this into the `types.ts`. But I must admit that I still miss experience about where to define enumerations, interfaces, types... in TypeScript. Just want to say that we should not start to mix these styles. So either adjust the others to do the same or export this from here.

Reply:
So I think the only types/enums which need to be exported in its own modules are those which may be reusable. We prefer to have single-use types local on where they're needed. The same goes for helper/utility functions: if they're generic and/or used multiple times, they can be split in a specific module, but IMO it's more readable to not need to be jumping back and forth on other files if a function/type/enum is only used and maybe only relevant in that context, than to split it by "type"... this is more of a Domain Driven Development, which we tend to follow on the light-client, as per @christianbrb 's lead. These types and functions belong to the same semantic domain. If it becomes too big, we can split, but the way we did with the epics, having them on separate files but domain-localized in a private module instead of the public/global ones.
I didn't want to refactor the others here, since I'm trying to do small PRs, but I'll make a follow up PR to adjust those.

Reply:
We agreed on the D3 approach in the daily call. :+1: